x-pack/test/cases_api_integration/common/lib/authentication/roles.ts

```diff
@@ -190,6 +190,29 @@ export const securitySolutionOnlyRead: Role = {
   },
 };
 
+export const securitySolutionOnlyReadAlerts: Role = {
```

Comment:
How about we create another user that does not have access to the elasticsearch indices and ensure that the user cannot update the status of an alert. We may already have a test that does that I can't remember.

Either way I think it'd be good to test adding the case ID to an alert when the security solution user does not have access to the elasticsearch index.